public/app/plugins/panel/timeline/module.tsx

```diff
@@ -17,6 +17,12 @@ export const plugin = new PanelPlugin<TimelineOptions, TimelineFieldConfig>(Time
     },
     useCustomConfig: (builder) => {
       builder
+        .addBooleanSwitch({
+          path: 'mergeValues',
```

Comment:
can mergeValues be a panel level setting applied at the top level?  I know that is a bigger refactor, so maybe in a follow up PR

Reply:
yes, maybe do the same for spanNulls? they have very similar semantics in that they work to interpolate across noisy data with lots of little gaps or lots of little same-state blips.